### PR TITLE
review messages-events-commands.md

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: nservicebus/messaging/messages-events-commands.md [hunk 1]
```diff
@@ -20,72 +20,65 @@ redirects:
...
 
+ ## Designing messages
 
-### Validation Messages
+ Messages represent data contracts between endpoints. They should be designed according to the following guidelines.
+ 
+ Messages should:
 
-There are checks in place to ensure following of the best practices. While violating above rules the following exceptions can be seen:
+ * be simple [POCO](https://en.wikipedia.org/wiki/Plain_old_CLR_object) objects.
```
Comment (kbaley) [inline nservicebus/messaging/messages-events-commands.md]: Having _objects_ at the end there is redundant but admittedly reads better

File: nservicebus/messaging/messages-events-commands.md [hunk 2]
```diff
@@ -20,72 +20,65 @@ redirects:
...
 public class MyCommand : ICommand { }
 
 public class MyEvent : IEvent { }
 
 public interface MyEvent : IEvent { }
+
+public class MyMessage : IMessage { }
 ```
 
+NOTE: This approach has some drawbacks, as described in ["Unobtrusive mode messages"](unobtrusive-mode.md).
```
Comment (kbaley) [inline nservicebus/messaging/messages-events-commands.md]: We don't normally use quotes when referencing other articles. Is this a new convention?

File: nservicebus/messaging/messages-events-commands.md (hunk 1 above)
Comment (adamralph) [reply nservicebus/messaging/messages-events-commands.md]: Yeah, true. That got carried over from the original text which I moved from below. How about "POCO types"?

File: nservicebus/messaging/messages-events-commands.md (hunk 2 above)
Comment (adamralph) [reply nservicebus/messaging/messages-events-commands.md]: 👍 I'll remove them.

**Comment by adamralph** [comment]
Based on @DavidBoike's 👍 to https://github.com/Particular/docs.particular.net/pull/4201#discussion_r244945934, I'm merging.

**Pull request merged**